extension/source/Background/index.ts

```diff
@@ -1,18 +1,18 @@
 import 'emoji-log';
 import { browser } from 'webextension-polyfill-ts';
-import { Aggregator } from 'bls-wallet-clients';
-import { initBlsWalletSigner } from 'bls-wallet-signer';
+import { Aggregator /* , initBlsWalletSigner */ } from 'bls-wallet-clients';
 import { ethers } from 'ethers';
 
 import App from '../App';
 import getPropOrUndefined from '../helpers/getPropOrUndefined';
 import RequestHandler from './RequestHandler';
 
-import { AGGREGATOR_URL, NETWORK_CONFIG, CHAIN_RPC_URL } from '../env';
+import { AGGREGATOR_URL /* , NETWORK_CONFIG */, CHAIN_RPC_URL } from '../env';
 
 (async () => {
   const app = new App(
-    await initBlsWalletSigner({ chainId: NETWORK_CONFIG.auxiliary.chainid }),
+    // TODO Is this still needed for something else?
+    // await initBlsWalletSigner({ chainId: NETWORK_CONFIG.auxiliary.chainid }),
```

Comment:
@voltrevo @jzaki Due to `BLSWalletWrapper` now wrapping the signer, this does not appear to be needed anymore. Safe to remove?

Reply:
Yeah that makes sense. Safe to remove 👍.

Reply:
```suggestion
```